i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md

```diff
@@ -160,6 +191,6 @@ yum install centreon-pack-operatingsystems-windows-nsclient-05-nrpe
 | X         | NRPETIMEOUT      | 30                        | Timeout value (Default: '30')                                    |
 | X         | NRPEEXTRAOPTIONS | **-u -2 -P 8192**         | Extraoptions to use with the NRPE binary (default: '-u -m 8192') |
 
-## Troubleshooting
+## Dépannage
 
-Please find the troubleshooting documentation for NRPE checks in the [dedicated chapter](../getting-started/how-to-guides/troubleshooting-plugins.md#nrpe-checks) of the Centreon documentation.
+Veuillez trouver la documentation de dépannage pour les contrôles NRPE dans le [chapitre dédié](../démarrer/guides/troubleshooting-plugins.md#nrpe-checks) de la documentation Centreon.
```

Comment:
It's not necessary to translate link here. Use the one from the EN part.

Reply:
ok, fixed